pages/vtransfer.vue

```diff
@@ -30,8 +33,8 @@
                 min="0"
                 :max="amount"
                 :disabled="amount == -1"
-                placeholder="0.0001"
-                step="0.0001"
+                placeholder="1.0000"
+                step="1"
```

Comment:
@djmbritt by putting step to 1 it means you can only input whole numbers, I guess we also want to allow floating numbers right?